Packs/QualysFIM/pack_metadata.json

```diff
@@ -2,7 +2,7 @@
     "name": "QualysFIM",
     "description": "Cloud solution for detecting and identifying critical changes, incidents, and risks resulting from normal and malicious events",
     "support": "xsoar",
-    "currentVersion": "1.0.33",
+    "currentVersion": "1.0.34",
```

Comment:
1.1.0

Reply:
Fixed in commit c264c71294807bac2a29bc298d788fb475d0dd1b